support/signal.c

```diff
@@ -290,11 +391,11 @@ Mono_Unix_UnixSignal_uninstall (void* info)
 		if (h->have_handler && count_handlers (h->signum) == 1) {
 			mph_sighandler_t p = signal (h->signum, h->handler);
 			if (p != SIG_ERR)
-				r = 0;
+				r = 0; // Success
```

Comment:
Needless comment; context (e.g. the default value being -1), along with use in `UnixSignal`, show that this method is using Unix semantics -- -1 on error, 0 on success.